tests/index.test.ts

```diff
@@ -45,12 +57,38 @@ describe('sdk tests', () => {
       runtime.application = originalRuntime.application;
       runtime.origin = originalRuntime.origin;
 
+      window.clearTimeout = originalClearTimeout;
       window.setTimeout = originalTimeout;
       window.addEventListener = originalEventListener;
       oauthService.openPopup = originalOpenPopup;
       jest.restoreAllMocks();
     });
 
+    describe('Outreachcontext initialized', () => {
+      it('will resolve context promise when success', async () => {
+        var message = new InitMessage();
+        message.result = '200';
+        message.application = new Application();
+        message.application.store = new ManifestStore();
+        message.context = [
```

Comment:
can we include all parameters that are described here? https://developers.outreach.io/client-extensions/javascript-sdk/#email 
Also the docs description of email context parameters `subj` and `to` does not correspond to the implementation. Please fix either here or in the docs (preferably).

Reply:
Yes.
I will fix the docs. `To` should be an array in docs. Subject states "message subject", what's wrong about that?

Reply:
In the docs it is called `subj` and the key name on the context object according to the test is `subject`

Reply:
That is correct. In `OutreachContext` we use full names. In query params we use `eml.subj`

Reply:
But the docs are about OutreachContext object and not about the query parameters, so I guess we need to fix the docs.